clang/lib/Sema/SemaCodeComplete.cpp

```diff
@@ -5674,6 +5675,11 @@ QualType getApproximateType(const Expr *E) {
         return getApproximateType(VD->getInit());
     }
   }
+  if (const auto *UO = llvm::dyn_cast<UnaryOperator>(E)) {
+    if (UO->getOpcode() == UnaryOperatorKind::UO_Deref) {
```

Comment:
nit: We [don't](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements) usually wrap a single statement with braces.